apps/api-journeys/src/app/modules/event/radioQuestion/radioQuestion.resolver.ts

```
@@ -4,27 +4,60 @@ import { UseGuards } from '@nestjs/common'
 import { Args, Mutation, Resolver } from '@nestjs/graphql'
 import { GqlAuthGuard } from '@core/nest/gqlAuthGuard/GqlAuthGuard'
 import { CurrentUserId } from '@core/nest/decorators/CurrentUserId'
+import { UserInputError } from 'apollo-server'
 import {
   RadioQuestionSubmissionEvent,
   RadioQuestionSubmissionEventCreateInput
 } from '../../../__generated__/graphql'
 import { EventService } from '../event.service'
+import { BlockService } from '../../block/block.service'
+import { VisitorService } from '../../visitor/visitor.service'
 
 @Resolver('RadioQuestionSubmissionEvent')
 export class RadioQuestionSubmissionEventResolver {
-  constructor(private readonly eventService: EventService) {}
+  constructor(
+    private readonly eventService: EventService,
+    private readonly blockService: BlockService,
+    private readonly visitorService: VisitorService
+  ) {}
+
   @Mutation()
   @UseGuards(GqlAuthGuard)
   async radioQuestionSubmissionEventCreate(
     @CurrentUserId() userId: string,
     @Args('input')
     input: RadioQuestionSubmissionEventCreateInput
   ): Promise<RadioQuestionSubmissionEvent> {
+    const block: { journeyId: string } = await this.blockService.get(
+      input.blockId
+    )
+    const journeyId = block.journeyId
```

need to throw a UserInput Error here if no block.